src/atomate2/common/jobs/phonons.py

```diff
@@ -57,32 +57,46 @@ def get_total_energy_per_cell(
 
 @job
 def get_supercell_size(
-    structure: Structure, min_length: float, prefer_90_degrees: bool, **kwargs
+    structure: Structure,
+    min_length: float,
+    max_length: float,
+    prefer_90_degrees: bool,
+    allow_orthorhombic: bool = False,
+    **kwargs,
 ) -> list[list[float]]:
     """
-    Determine supercell size with given min_length.
+    Determine supercell size with given min_length and max_length.
 
     Parameters
     ----------
     structure: Structure Object
         Input structure that will be used to determine supercell
     min_length: float
         minimum length of cell in Angstrom
+    max_length: float
+        maximum length of cell in Angstrom
     prefer_90_degrees: bool
         if True, the algorithm will try to find a cell with 90 degree angles first
+    allow_orthorhombic: bool
+        if True, orthorhombic supercells are allowed
     **kwargs:
         Additional parameters that can be set.
     """
     kwargs.setdefault("force_diagonal", False)
-    common_kwds = {
-        "min_length": min_length,
-        "min_atoms": kwargs.get("min_atoms"),
-        "force_diagonal": kwargs["force_diagonal"],
-    }
+    common_kwds = dict(
+        min_length=min_length,
+        max_length=max_length,
+        min_atoms=kwargs.get("min_atoms"),
+        force_diagonal=kwargs["force_diagonal"],
+        step_size=kwargs["step_size"],
+    )
 
     if not prefer_90_degrees:
         transformation = CubicSupercellTransformation(
-            **common_kwds, max_atoms=kwargs.get("max_atoms"), force_90_degrees=False
+            **common_kwds,
+            max_atoms=kwargs.get("max_atoms"),
```

Comment:
why is this not part of theother dict?

Reply:
It was like this in the original implementation, so I left it like that.

Reply:
Okay, fine

Reply:
If you would like me to, I could move it to common_kwds as I don't think it will cause any backwards incompatibility issues and I would find it more systematic to have all kwargs in common_kwds.

Reply:
Sounds good

Reply:
I forgot to push those changes, I notice now, but it doesn't affect the functionalities.

Reply:
I finally thought of pushing this change 😅